docs/diagrams/EntityCommandSequenceDiagram.puml

```diff
@@ -0,0 +1,57 @@
+@startuml
+!include style.puml
+skinparam ArrowFontStyle plain
+
+box Logic LOGIC_COLOR_T1
+participant ":LogicManager" as LogicManager LOGIC_COLOR
+participant ":AddressBookParser" as AddressBookParser LOGIC_COLOR
+participant ":xyzCommandParser" as CommandParser LOGIC_COLOR
+participant "e:XYZCommand" as XYZCommand LOGIC_COLOR
+participant "r:CommandResult" as CommandResult LOGIC_COLOR
+end box
+
+[-> LogicManager : execute("xyz")
+activate LogicManager
+
+create CommandParser
+activate AddressBookParser
+AddressBookParser -> CommandParser
+activate CommandParser
+AddressBookParser <-- CommandParser
+deactivate CommandParser
```

Comment:
Should we create the `CommandParser` only after we invoke the `parseCommand("xyz")` method? Perhaps you can move this code down to after the line `LogicManager -> AddressBookParser : parseCommand("xyz")`